.github/workflows/stackhawk.yml

```
@@ -0,0 +1,63 @@
+  pull_request:
+    branches: [ "main" ]
+  schedule:
+    - cron: '36 21 * * 1'
+
+permissions:
+  contents: read
+
+jobs:
+  stackhawk:
+    permissions:
+      contents: read # for actions/checkout to fetch code
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)